<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParagraphIndentSpacingTab</class>
 <widget class="QWidget" name="ParagraphIndentSpacingTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>328</height>
   </rect>
  </property>
  <property name="whatsThis">
   <string>&lt;p&gt;&lt;ul&gt;&lt;li&gt;&lt;b&gt;Single&lt;/b&gt;: The normal linespacing&lt;/li&gt;&lt;/ul&gt;&lt;/p&gt; </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="indentGroupBox">
     <property name="title">
      <string>Indent</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" alignment="Qt::AlignRight">
       <widget class="QPushButton" name="leftLabel">
        <property name="text">
         <string>Left:</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KoUnitDoubleSpinBox" name="left"/>
      </item>
      <item row="0" column="3" rowspan="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" alignment="Qt::AlignRight">
       <widget class="QPushButton" name="rightLabel">
        <property name="text">
         <string>Right:</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KoUnitDoubleSpinBox" name="right"/>
      </item>
      <item row="2" column="0" alignment="Qt::AlignRight">
       <widget class="QPushButton" name="firstLineLabel">
        <property name="text">
         <string>First Line:</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KoUnitDoubleSpinBox" name="first"/>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="autoTextIndent">
        <property name="text">
         <string>Auto-infer from text size</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="lineSpacingGroupBox">
     <property name="whatsThis">
      <string>&lt;p&gt;&lt;ul&gt;&lt;li&gt;&lt;b&gt;Single&lt;/b&gt;: The normal linespacing&lt;/li&gt;&lt;/ul&gt;&lt;/p&gt; </string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="lineSpacingLabel">
        <property name="text">
         <string>Line Spacing</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" stretch="0,0,1">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="lineSpacing"/>
        </item>
        <item>
         <widget class="QStackedWidget" name="spacingStack">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="unitsPage">
           <layout class="QHBoxLayout">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="KoUnitDoubleSpinBox" name="custom"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="percentPage">
           <layout class="QHBoxLayout">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QSpinBox" name="proportional">
              <property name="suffix">
               <string> %</string>
              </property>
              <property name="minimum">
               <number>85</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
              <property name="singleStep">
               <number>5</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="1" rowspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>173</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <layout class="QFormLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Use Font Metrics:</string>
          </property>
          <property name="buddy">
           <cstring>useFont</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="useFont">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="paragraphSpGroupBoxacing">
     <property name="title">
      <string>Paragraph Space</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QPushButton" name="beforeLabel">
        <property name="text">
         <string>Before:</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KoUnitDoubleSpinBox" name="before"/>
      </item>
      <item row="0" column="2" rowspan="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>213</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="afterLabel">
        <property name="text">
         <string>After:</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KoUnitDoubleSpinBox" name="after"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>313</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KoUnitDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>KoUnitDoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
